VPC peering article on GCP

dewan-ahmed opened the pull request:
# What changed, and why it matters

This is a migration related PR that resolves https://github.com/aiven/devportal/issues/1105

Original help article is https://help.aiven.io/en/articles/778836-using-virtual-private-cloud-vpc-peering that has been broken into four articles. The general VPC peering article is live and this one brings in the VPC peering steps for GCP. 

I have validated the steps for GCP. Please update relevant migration sheets after reviewing and merging.

staceysalamon-aiven commented on docs/platform/howto/manage-vpc-peering.rst:
```diff
@@ -30,6 +30,11 @@ When you create a new service, you can then place it in the VPC. The **VPC** tab
+Cloud-specific VPC peering instructions
+-----------------------------------------------------
+
+- Refer to :ref:`GCP VPC peering instructions <vpc-peering-gcp>`.
```
```suggestion
- :ref:`Set up VPC peering on Google Cloud Platform (GCP) <vpc-peering-gcp>`.
```

staceysalamon-aiven replied:
Small suggestion to spell out GCP and make the language more consistent with the page the user is navigating to.

staceysalamon-aiven commented on docs/platform/howto/manage-vpc-peering.rst:
```diff
@@ -30,6 +30,11 @@ When you create a new service, you can then place it in the VPC. The **VPC** tab
    .. note::
```
```suggestion
.. note::
```

staceysalamon-aiven replied:
Just moving the indent to remove the block quote formatting/grey background.

staceysalamon-aiven commented on docs/platform/howto/vpc-peering-gcp.rst:
```diff
@@ -0,0 +1,30 @@
+.. _vpc-peering-gcp:
+
+Set Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)
```
```suggestion
Set up Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)
```

staceysalamon-aiven commented on docs/platform/howto/vpc-peering-gcp.rst:
```diff
@@ -0,0 +1,30 @@
+Set Virtual Private Cloud (VPC) peering on Google Cloud Platform (GCP)
+======================================================================
+
+Once you've created a :ref:`VPC on the Aiven platform <platform_howto_setup_vpc_peering>`, you can follow this instruction to set up VPC peering on AWS.
```
```suggestion
Once you've created a :ref:`VPC on the Aiven platform <platform_howto_setup_vpc_peering>`, you can follow these instructions to set up VPC peering on GCP.
```

staceysalamon-aiven commented on docs/platform/howto/vpc-peering-gcp.rst:
```diff
@@ -0,0 +1,30 @@
+
+1. Open your GCP Console.
+
+2. Select **VPC Networks** on the left and find the VPC that you want to connect.
```
```suggestion
2. Select **VPC Networks** and find the VPC that you want to connect to.
```

staceysalamon-aiven replied:
I would suggest removing "on the left" from here, if reasonable, to avoid needing to check/update things later if GCP changes their console UI.

staceysalamon-aiven commented on docs/platform/howto/vpc-peering-gcp.rst:
```diff
@@ -0,0 +1,30 @@
+8. Enter a name for the peering connection and enter the same project ID and network name that are shown in the Aiven web console.
+
+   .. note::
+       You can click the blue Pending peer icon to see the Aiven project name and the network name.
```
Super minor point, but in step 6 Pending Peer uses title case. :)

staceysalamon-aiven replied:
Would this be better as step 7? The same way we guide the users to make note of the GCP info in steps 3 and 4, we can tell them to get the project ID and network name before creating the connection in GCP. What do you think?

staceysalamon-aiven reviewed (changes requested):
I made a few small suggestions. Let me know if anything's unclear! :)

dewan-ahmed commented:
Thank you @staceysalamon-aiven for reviewing. I've added the suggestion and moved the note to step6 for ease of following for the reader. Please take one more look when you get a chance.

staceysalamon-aiven commented:
Great! Looks good to me!
